sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.4.0b2 (Unreleased)
 
 ### Features Added
+- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
```

Comment:
Add the same line to the other SDKs in which the identifier will be exposed

Reply:
Added.